### DISCUSSION TITLE
Booting from iso image stops on a black screen with blinking cursor

### DISCUSSION THREAD

**Discussion by sikelos** [Security-Onion-Solutions/securityonion#8292] [Unsupported Versions] [answered]
I am using open nebula KVM hypervisor. I have used similar boot procedures with several iso files many times with no Issues.

I have downloaded 2 times the security onion iso file for version 2.3.130 
trying to boot from iso: it loads a couple of pages with log/booting information and then on next stage loads a black screen with a blinking cursor and stops there.
Same issue with both downloaded iso files.

Thanks a lot to any suggestions or similar issues using iso installation.

![image](https://user-images.githubusercontent.com/25420601/178778270-dd5e48c8-e490-4d68-b6ce-8324157912c8.png)

**Comment by sikelos** [comment, marked as answer]
After many trial and error tests, found the issue: The attached empty disk I was using to install the SO, was flagged on creation (in Open Nebula) with the RAW driver type. You need to use the QCOW2 driver type and then the installation goes further.

Possibly the SO iso installation stops when a RAW storage device is found attached but no error is given. ISO needs to be improved so it notifies about this blocking factor.

**Reply by dougburks** [reply]
Looks like issue https://github.com/Security-Onion-Solutions/securityonion/issues/1825 may be applicable.
